Extension/src/common.ts

```diff
@@ -912,6 +912,26 @@ export async function promptReloadWindow(message: string): Promise<void> {
     }
 }
 
+export async function promptCDDFailed(copy?: boolean): Promise<void> {
+    const copyMessage: string = localize("cdd.copyMessage", "Failed to copy declaration or definition to clipboard.");
+    const createMessage: string = localize("cdd.createMessage", "Failed to create declaration or definition.");
```

Comment:
For what scenario is the error `createMessage` used for? The error messages for the operation "create declaration/definition" is set in the native side. There are also strings already defined for this too.

Reply:
Is it possible for CDD to fail after a successful request to the native side?